docs/lib/usage/starlite-app/app-layers.rst

```diff
@@ -0,0 +1,34 @@
+Layered Architecture
+====================
+
+.. TODO: Elaborate this section and reorganise the hyperlinks to the rest of the docs.
```

Comment:
Can we elaborate on this section a bit more? For example, what do we exactly mean by "_the layered architecture_" & how does it differ from other similar API frameworks?

I think the readers will appreciate a bit more context here.